daemon/service-db.cc

```diff
@@ -279,27 +274,18 @@ MLServiceDB::get_pipeline (const std::string name, std::string &description)
   std::string key_with_prefix = DB_KEY_PREFIX;
   key_with_prefix += name;
 
```

Comment:
How about add 
```
  if (name.empty () || description.empty ())
    throw std::invalid_argument ("Invalid name or value parameters!");
```

Reply:
`description` is reference of std::string. I think empty check of it is unnecessary.